features/step_definitions/machine.rb

```diff
@@ -84,6 +84,16 @@
   cb["new_node"] = node_name
 end
 
+Then(/I check that cluster does not have empty publiczone$/) do
+  ensure_admin_tagged
+
+  if dns("cluster").publicZone == nil
+     logger.warn "PublicZone is empty, default values won't work"
+     logger.warn "We will skip this scenario"
```

Comment:
Instead of saying `We will skip this scenario`, please say `The scenario will be skipped because ...` hereafter. Compress the two lines of `logger.warn` into one line.